File: mekonnen-naod/Calculator-Coin-Combination.js
// return a combination of the smallest possible combination of coins for the cent value
// input:cent amount
// output:an array of coin combinations equaling the given cent value
// still not sure how this works

function coinCombo (cents) {
	let centsIn = cents;
  let totalOut = [];
  let quarters = Math.floor(centsIn/25);
  centsIn = centsIn - (quarters*25);
  
  let dimes = Math.floor(centsIn/10);
  centsIn = centsIn - (dimes*10);
  
  let nickels = Math.floor(centsIn/5);
  centsIn = centsIn - (nickels*5);
  
  let pennies = centsIn;
  pennies == centsIn - (pennies);
  
  return [pennies, nickels, dimes, quarters];
}

File: mekonnen-naod/Calculator-Coin-Combination.py
import math 
def coin_combo(cents):
    cents_in = cents
    
    quarters = cents_in//25
    cents_in = cents_in - (quarters*25)
    
    dimes = cents_in//10
    cents_in = cents_in - (dimes*10)
    
    nickels = cents_in//5
    cents_in = cents_in - (nickels*5)
    
    pennies = math.floor(cents_in)
    cents_in = cents_in - (pennies)
    
    return [pennies, nickels, dimes, quarters]
